tests/cache/cache.test.js

```diff
@@ -1,83 +0,0 @@
-const lruCache = require('lru-cache');
-const Redis = require('ioredis-mock');
-const logger = require('../../src/utils/logging');
-const cache = require('../../src/cache');
-
-const redis = new Redis({
-  data: {
-    existInRedis: JSON.stringify({ result: 'valueInRedis' }),
-  },
-});
-
-cache.redisClient.master.redis = redis;
-cache.redisClient.slave.redis = redis;
-
-jest.mock('../../src/utils/logging');
-
-jest.mock('../../src/cache/redis-client', () => jest.fn(() => ({
-  createClient: jest.fn(),
-  master: {
-    isHealthy: true,
-    checkStatus: jest.fn(),
-  },
-  slave: {
-    isHealthy: true,
-    checkStatus: jest.fn(),
-  },
-})));
-
-jest.mock('lru-cache', () => jest.fn().mockImplementation(() => ({
-  get: jest.fn((key) => {
-    if (key === 'existInL1') return { result: 'valueInL1' };
-    return null;
-  }),
-  set: jest.fn(),
-})));
-
-jest.mock('../../src/cache/aws-utilities', () => jest.fn((value) => new Promise((resolve, reject) => {
-  if (value === 'placement/availability-zone') {
-    resolve('eu-west-1b');
-  } else {
-    reject(new Error('error'));
-  }
-})));
```

Comment:
All of this is replaced by the real deal (e.g. with `lru`), or a mocked library instead of a manual mock.

See `index-l1-hot.js`.